src/main/webapp/app/exercises/shared/result/result.component.ts

```diff
@@ -138,7 +174,9 @@ export class ResultComponent implements OnInit, OnChanges {
     evaluate() {
         this.templateStatus = this.evaluateTemplateStatus();
 
-        if (this.templateStatus === ResultTemplateStatus.LATE) {
+        if (this.templateStatus === ResultTemplateStatus.MISSING) {
```

Comment:
I'd suggest removing this and replacing the `else` clause with an `else if (this.templateStatus !== ResultTemplateStatus.MISSING)`

Reply:
Yes, we can change that.